### ISSUE TITLE
Reset brick light (ev3dev) does not what is written in the tooltip

### ISSUE THREAD

**Issue by VinArt** [OpenRoberta/openroberta-lab#242]
- reset brick light does not reset the led to blinking green, it simply turns off the led. 
- tooltip and help files do not match

this could be caused by different default behaviour of lejos and dev firmware, this should be unified and help files and tooltips are changed to be matching.

**Comment by ensonic** [comment]
Indeed. On ev3dev the leds blink green while booting, but afterwards they are off.
